x-pack/plugins/security_solution/public/common/components/events_tab/events_query_tab_body.tsx

```diff
@@ -66,12 +67,16 @@ const EventsQueryTabBodyComponent: React.FC<EventsQueryTabBodyComponentProps> =
   startDate,
   timelineId,
 }) => {
+  const isEnterprisePlus = useLicense().isEnterprise();
   const dispatch = useDispatch();
   const { globalFullScreen } = useGlobalFullScreen();
   const tGridEnabled = useIsExperimentalFeatureEnabled('tGridEnabled');
   const [defaultNumberFormat] = useUiSetting$<string>(DEFAULT_NUMBER_FORMAT);
   const [showExternalAlerts, setShowExternalAlerts] = useState(false);
-  const leadingControlColumns = useMemo(() => getDefaultControlColumn(ACTION_BUTTON_COUNT), []);
+  const leadingControlColumns = useMemo(
+    () => getDefaultControlColumn(isEnterprisePlus ? ACTION_BUTTON_COUNT : 4),
```

Comment:
would you mind moving the const ACTION_BUTTON_COUNT down into this component and then following the same paradigm in the other places of 
```
  const isEnterprisePlus = useLicense().isEnterprise();
  const ACTION_BUTTON_COUNT = isEnterprisePlus ? 5 : 4;
```